### PR TITLE
Remove EbrCenterTextInRectVertically

### LINKED ISSUE #1812: UITextView used EbrCenterTextInRectVertically; figure out why.
This was removed in CGD2D, as EbrCenterTextInRectVertically was living in CGContext. It might be more of a CoreText concern.

### PR DESCRIPTION
**Pull request by aballway**
Changes UITabBarButton to directly center text vertically in drawRect:
Removes dead code from UITextView-sizeThatFits:

Fixes #1812

### CODE REVIEW

**Comment by jaredhms** [comment]
:shipit:

File: Frameworks/UIKit/UITabBarButton.mm [hunk 1]
```diff
@@ -126,15 +126,12 @@ - (void)drawRect:(CGRect)pos {
         id font = [UIFont defaultFont];
         size = [title sizeWithFont:font constrainedToSize:CGSizeMake(0.0f, 0.0f) lineBreakMode:UILineBreakModeClip];
 
-        CGRect textRect;
-        textRect.origin.y = rect.size.height - size.height;
+        CGRect textRect{};
+        textRect.origin.y = (rect.size.height - size.height) / 2.0;
```
Comment (jaredhms) [inline Frameworks/UIKit/UITabBarButton.mm]: probably worth commenting that we're vertically centering this

File: Frameworks/UIKit/UITabBarButton.mm [hunk 2]
```diff
@@ -127,6 +127,8 @@ - (void)drawRect:(CGRect)pos {
         size = [title sizeWithFont:font constrainedToSize:CGSizeMake(0.0f, 0.0f) lineBreakMode:UILineBreakModeClip];
 
         CGRect textRect{};
+
+        // Horizontally center text
```
Comment (jaredhms) [inline Frameworks/UIKit/UITabBarButton.mm]: vertically?

Comment (aballway) [inline Frameworks/UIKit/UITabBarButton.mm]: Yes, thank you 😪

Comment (jaredhms) [inline Frameworks/UIKit/UITabBarButton.mm]: double sign off!

---
In reply to: [98794256](https://github.com/Microsoft/WinObjC/pull/1882#discussion_r98794256) [](ancestors = 98794256)

File: Frameworks/UIKit/UITabBarButton.mm [hunk 3]
```diff
@@ -126,15 +126,14 @@ - (void)drawRect:(CGRect)pos {
         id font = [UIFont defaultFont];
         size = [title sizeWithFont:font constrainedToSize:CGSizeMake(0.0f, 0.0f) lineBreakMode:UILineBreakModeClip];
 
-        CGRect textRect;
-        textRect.origin.y = rect.size.height - size.height;
+        CGRect textRect{};
```
Comment (DHowett-MSFT) [inline Frameworks/UIKit/UITabBarButton.mm]: If you're gonna do this, y'might'swell fold the following statements into the initializer.

**Pull request merged**